src/Approximations/box_approximations.jl

```diff
@@ -1,207 +1,223 @@
-        hbottom = -ρ(dminus, X)
-        dplus[i] = 0.0
-        dminus[i] = 0.0
-        c[i] = (htop+hbottom)/2.
-        r[i] = (htop-hbottom)/2.
+    d = zeros(n)
+    @inbounds for i in 1:n
+        d[i] = 1.
+        htop = ρ(d, S)
+        d[i] = -1.
+        hbottom = -ρ(d, S)
+        d[i] = 0.
+        c[i] = (htop + hbottom) / 2.
+        r[i] = (htop - hbottom) / 2.
     end
-    return n, c, r
+    return c, r
 end
 
 """
-    ballinf_approximation(X)
+    ballinf_approximation(S)
 
-Overapproximation of a set by a ball in the infinity norm.
+Overapproximate a convex set by a tight ball in the infinity norm.
 
 ### Input
 
-`X` -- a lazy set
+- `S` -- convex set
 
 ### Output
 
-`H` -- a ball in the infinity norm which tightly contains the given set
+A tight ball in the infinity norm.
 
 ### Algorithm
 
 The center and radius of the box are obtained by evaluating the support function
-of the given set along the canonical directions.
+of the given convex set along the canonical directions.
 """
-function ballinf_approximation(X::LazySet)::BallInf
-    n = dim(X)
-    c = Vector{Float64}(n) # TODO: get numerical eltype of X
+function ballinf_approximation(S::LazySet)::BallInf
+    n = dim(S)
+    c = Vector{Float64}(n) # TODO: get numerical eltype of S
     r = 0.
-    dplus = zeros(n)
-    dminus = zeros(n)
+    d = zeros(n)
     @inbounds for i in 1:n
-        dplus[i] = 1.0
-        dminus[i] = -1.0
-        htop = ρ(dplus, X)
-        hbottom = -ρ(dminus, X)
-        dplus[i] = 0.0
-        dminus[i] = 0.0
-        c[i] = (htop+hbottom)/2.
-        rcur = (htop-hbottom)/2.
+        d[i] = 1.
+        htop = ρ(d, S)
+        d[i] = -1.
+        hbottom = -ρ(d, S)
+        d[i] = 0.
+        c[i] = (htop + hbottom) / 2.
+        rcur = (htop - hbottom) / 2.
         if (rcur > r)
             r = rcur
         end
     end
     return BallInf(c, r)
 end
 
-# ========================================================
+# =======================================================
 # Metric properties of sets computed using Approximations
-# ========================================================
+# =======================================================
 
 """
-    norm(X::LazySet, [p])
+    norm(S::LazySet, [p]::Real=Inf)
 
-Return the norm of a `LazySet`. It is the norm of the enclosing ball (of
-the given norm) of minimal volume.
+Return the norm of a convex set.
+It is the norm of the enclosing ball (of the given norm) of minimal volume.
 
 ### Input
 
-- `X` -- a lazy set
+- `S` -- convex set
 - `p` -- (optional, default: `Inf`) norm
 
 ### Output
 
 A real number representing the norm.
 """
-function norm(X::LazySet, p::Real=Inf)
+function norm(S::LazySet, p::Real=Inf)
     if p == Inf
-        return norm(ballinf_approximation(X), p)
+        return norm(ballinf_approximation(S), p)
     else
-        error("The norm for this value of p=$p is not implemented")
+        error("the norm for this value of p=$p is not implemented")
```

Comment:
thanks for fixing this -- i added this [guideline in the wiki](https://github.com/JuliaReach/LazySets.jl/wiki/Documentation-Guidelines#more-guidelines-) some time ago

Reply:
Yes, I saw that (accidentally) 😀